talkative_robot_2.rb

```diff
@@ -0,0 +1,307 @@
+require 'pry'
+
+class Person
+  
+  attr_accessor :name, :gender, :age, :grocery_list
+  
+  def initialize(args)
+    @name   			= args[:name]   || nil
+    @gender 			= args[:gender] || nil
```

Comment:
`nil` can cause a bunch of errors. Better to give defaults similar to what you'd expect. In this case, an empty string may work. For name, "John Doe" would be OK.